snippets/abbreviations.md

```diff
@@ -12,6 +12,7 @@
 *[BAU]: Business as Usual
 *[SOP]: Standard Operating Procedures
 *[CMS]: Case Management System
+*[CIN]: **C**MS **IN**tegration
```

Comment:
Unfortunately the abbreviations do not render markdown, so you'll want to replace this with `Case Management System Integration` to make it work.

Also another unfortunate thing is that the abbreviations only work with a full word, so `CIN1` will not use this abbreviation, you'll want to add `*[CIN1]: Case Management System Integration One`

Reply:
Done